ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -95,7 +98,7 @@ objects:
       # we want to gather the teardown logs no matter what
       ci-operator.openshift.io/wait-for-container-artifacts: teardown
       ci-operator.openshift.io/save-container-logs: "true"
-      ci-operator.openshift.io/container-sub-tests: "setup,test,teardown"
+      ci-operator.openshift.io/container-sub-tests: "setup,test,teardown,setup-packet"
```

Comment:
It shouldn't have since it execution is guarded by a condition on the CLUSTER_TYPE (see https://github.com/derekhiggins/release/blob/44d62d646ac884ec448f9e0b36f216800af2fd53/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L1002) - same approach used on the other containers

Reply:
This [annotation determines which tests are returned as JUnit tests](https://github.com/openshift/ci-tools/blob/6720bbb096b8a4d40f2717dcf69f363a64360868/pkg/steps/artifacts.go#L41-L42) so setup-packet would be displayed as a test on every test, even those not using Packet.